package.json

```diff
@@ -9,7 +9,8 @@
     "format": "prettier --check . && eslint .",
     "format:fix": "prettier --write .",
     "lint": "eslint .",
-    "lint:fix": "eslint . --fix"
+    "lint:fix": "eslint . --fix",
+    "test:unit": "vitest --coverage"
```

Comment:
Let's add this command also `"test": "vitest run"`. It's seems to me convenient for usage.